styles.css

```diff
@@ -478,3 +527,246 @@ h4 {
   width: max-content;
   margin-bottom: 32px;
 }
+
+ul:nth-child(4) {
+  padding-left: 0;
+}
+
+/* ------------ DESKTOP scaling ------------ */
+
+@media screen and (min-width: 768px) {
+  .hdr {
+    position: fixed;
+    width: 100%;
+    z-index: 999;
+    padding-top: 15px;
+    padding-left: 133px;
+    margin-top: 0;
+  }
+
+  header {
+    border-bottom-left-radius: 130px;
+  }
+
+  .main-body {
+    background-image: url(images/desktop-header-bg.png);
+  }
+
+  #div1-bgr {
+    background-image: url(images/desktop-div1-bg.png);
+  }
+
+  #div2-bgr {
+    background-image: url(images/desktop-div2-bg.png);
+  }
+
+  #div3-bgr {
+    background-image: url(images/desktop-div3-bg.png);
+  }
+
+  #div4-bgr {
+    background-image: url(images/desktop-div4-bg.png);
+  }
+
+  .par,
+  .h1 {
+    width: 100%;
+  }
+
+  #div1-bgr,
+  #div2-bgr,
+  #div3-bgr,
+  #div4-bgr {
+    background-size: contain;
+    height: auto;
+    width: 50%;
+    max-width: 568px;
+  }
+
+  .bar {
+    display: none;
+  }
+
+  .bar-desktop {
+    display: block;
+    margin: 0 142px auto auto;
+  }
+
+  .bar-desktop-list {
+    display: flex;
+    list-style: none;
+    justify-content: flex-start;
+    gap: 10px;
+  }
+
+  .bdl-el {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    padding: 10px 12px;
+  }
+
+  .h1-text {
+    margin: 288px 260px 0 260px;
+    padding: 0;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  .par1 {
+    margin: 0 260px 0 260px;
+    padding: 12px 0 12px 0;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .h2-text {
+    margin: 0 260px 14px 260px;
+    padding: 0;
+  }
+
+  .ulist {
+    margin: 0 1010px 255px 260px;
+    padding: 0;
+  }
```

Comment:
- [ ] Please refer to the attached image. Your webpage still lacks responsiveness on medium screens. This can be easily fixed by using flexbox instead of using margin to properly align elements.

![image](https://user-images.githubusercontent.com/92215878/186737121-e216d5a0-b9ef-4d3a-8761-6111c0a6e6a0.png)